src/app/app-modules/core/components/app-header/app-header.component.ts

```
@@ -100,6 +107,23 @@
     // }
     if (this.isAuthenticated) {
       this.fetchLanguageSet();
+      this.auth.getUserDetails().subscribe((res: any) => {
+        // console.log(">>>>>>", res);
+        if (res.statusCode == '200') {
+          if (res.data.previlegeObj && res.data.previlegeObj[0]) {
+            this.cookieService.set('Jwttoken', res.data.Jwttoken);
+            delete res.data.Jwttoken;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)